generator/interface_loader.go

```diff
@@ -47,6 +47,7 @@ func methodForSignature(sig *types.Signature, methodName string, imports Imports
 	}
 	return Method{
 		Name:    methodName,
+		Title:   strings.Title(methodName),
```

Comment:
Instead of storing this value as a field I think it would make sense to add `strings.Title` to the list of functions available to the template (ex: `interfaceFuncs` in `interface_template.go`).

That way the templates could use `{{ title .Name }}` and there would be no need for the extra field.